AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/PrimaryKey/README.md

```diff
@@ -0,0 +1,44 @@
+# DataStore Primary Key Integration Tests
+
+The following steps demonstrate how to setup a GraphQL endpoint with AppSync and IAM.
+This configuration is used to run the tests in `AWSDataStorePrimaryKeyTest.swift`.
+
+### Set-up
+
+1. `amplify init`
```

Comment:
let's document the `amplify --v` version that was used here and that it is V2 transform